networks/devnet.nix

```diff
@@ -67,21 +67,15 @@
             staking.params = {
               epoch_length = "8";
               jailed_validator_threshold = "10";
+              unbonding_time = "2m";
```

Comment:
still needed?

Reply:
I have no preference but I think it's fine to keep it as is on devnet

Reply:
ah, i thought this was in the cosmwasm deployer file. agreed 2mins is good 👍